tests/tensor/test_blas.py

```diff
@@ -1722,7 +1723,7 @@ def setup_method(self):
 
         self.fa = tt.fscalar()
         self.da = tt.dscalar()
-        self.ca = tt.cscalar()
+        self.ca = theano.tensor.type.cscalar()
```

Comment:
This should just be `tt.type.cscalar()`.

Reply:
It doesnt, but I added the import